ci-operator/step-registry/cucushift/installer/rehearse/vsphere/ipi/ovn/local-to-shared-gateway-mode-migration/cucushift-installer-rehearse-vsphere-ipi-ovn-local-to-shared-gateway-mode-migration-workflow.metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+	"path": "cucushift/installer/rehearse/vsphere/ipi/ovn/local-to-shared-gateway-mode-migration/deprovision/cucushift-installer-rehearse-vsphere-ipi-ovn-local-to-shared-gateway-mode-migration-deprovision-chain.yaml",
```

Comment:
Seems the path is incorrect, this is should be `...-workflow.yaml` instead of `-deprovision-chain`.